Visitor/Arithmetic.php

```diff
@@ -383,7 +383,14 @@ protected function initializeConstants ( ) {
                 'SQRT_PI' => M_SQRTPI,
                 'SQRT_2'  => M_SQRT2,
                 'SQRT_3'  => M_SQRT3,
-                'LN_PI'   => M_LNPI
+                'LN_PI'   => M_LNPI,
+                'LOG2E'   => M_LOG2E,
```

Comment:
If we follow the existing “logic”, we should have `LOG_2_E` instead of `LOG2E`, or maybe `LOG_2E`.